core/src/main/scala/kafka/zookeeper/ZooKeeperClient.scala

```diff
@@ -442,45 +493,51 @@ sealed trait AsyncRequest {
   type Response <: AsyncResponse
   def path: String
   def ctx: Option[Any]
+  def checkInfo: Option[ZkVersionCheck]
```

Comment:
Would it be simpler to name it `zkVersionCheck`?

Reply:
Yes. Done.